src/ApiIncrementalChecker.ts

```diff
@@ -129,7 +152,11 @@ export class ApiIncrementalChecker implements IncrementalCheckerInterface {
           this.tsIncrementalCompiler.getProgram()
         );
         // const source = fs.readFileSync(updatedFile, 'utf-8');
-        linter.lint(updatedFile, undefined!, this.linterConfig);
+        linter.lint(
+          updatedFile,
+          undefined!,
+          this.linterConfig || this.getLinterConfig(updatedFile)
```

Comment:
Can you talk me through this line please? What is `this.linterConfig` in this context? What would cause it to be `false-y` / when would you expect to call `this.getLinterConfig(updatedFile)`

Reply:
I tried to be more explicit this time around, but still: `this.linterConfig` - is a TSLint config, which is defined by path, provided by user in webpack config. Conversely, when user provides `true` instead of config-file path, `this.linterConfig` should be `undefined`, instead, it will be calculated inside `this.getLinterConfig`